Service/Suggestion/IndexBuilder.php

```diff
@@ -102,19 +113,22 @@ protected function rebuildStoreSuggestionIndexPage(int $storeId, QueryCollection
         $collection = clone $collectionDefault;
         $collection->setCurPage($page)->setPageSize($pageSize);
         $collection->load();
-        $indexName = $this->suggestionHelper->getTempIndexName($storeId);
+        $indexOptions = $this->indexOptionsBuilder->buildWithComputedIndex(
```

Comment:
Can use an `EntityIndexOptionsBuilderInterface` implementation for suggestion entities.